test/test_evaluator.cpp

```diff
@@ -274,75 +269,6 @@ TEST_F(ExprEvalTest, PostLower) {
   checkIntValue(evaluator, tid_x, 128);
 }
 
-// Kernel IR: Evaluate basic scalar operations with constant values
-TEST_F(ExprEvalTest, KernelConstants) {
```

Comment:
In the past, we had two copy-pasted expr evaluators for fusion and kernel. Now we only have one, so the copy-pasted tests should be removed.